src/services/user.ts

```diff
@@ -48,6 +55,11 @@ async function getMany(options: PaginationOptions) {
   );
 }
 
+async function getUserList(params: UserListParams) {
+  const users = await clerkClient.users.getUserList(params);
+  return users.map((user) => toResponse(user));
+}
+
```

Comment:
This looks like a function with duplicated implementation. You can probably just use `userMapFromIds`or the `getMany` function below.